Entities/ACRocket.cpp

```diff
@@ -861,9 +849,7 @@ void ACRocket::Update()
     // LOWER the gears
     else if (m_pMThruster && !m_pMThruster->IsEmitting())// && m_GearState != LOWERED)
     {
-/* Obsolete way of adding and removing atoms to the body group
-        m_pAtomGroup->RemoveAllButAtoms(0);
-
+/*
```

Comment:
I'm going to remove this code, it's not directly related to this issue but it's obsolete stuff from when footgroups were put in as atoms directly I think.

Reply:
You were going to but you didn't?

Reply:
Sometimes you have to refresh a few times I find. It's weird.